R/database.R

```diff
@@ -2695,19 +2641,35 @@ get_manta_sv_by_samples = function(these_samples_metadata,
     dplyr::filter(need_lift == TRUE)
   
   #lift to selected projection
-  lifted_calls = liftover_bedpe(bedpe_df = to_be_lifted, target_build = projection, print_head = FALSE)
+  lifted_calls = liftover_bedpe(bedpe_df = to_be_lifted, target_build = projection)
   
   #subset calls that does not need a "lift"
   no_lift_needed = merged_bedpe %>%
     dplyr::filter(need_lift == FALSE)
   
   #combine calls (lifted and not lifted), arrange and sort accordingly, drop temporary column
-  merged_bedpe = rbind(lifted_calls, no_lift_needed) %>%
-    arrange(CHROM_A, CHROM_B, START_A) %>%
+  merged_bedpe = bind_rows(lifted_calls, no_lift_needed) %>%
     dplyr::select(-need_lift)
-  
+
+  #Deal with chr prefixes based on the selected projection.
+  if(projection %in% c("hg38", "hg19")){
+    merged_bedpe = merged_bedpe %>%
+      dplyr::mutate(CHROM_A = case_when(str_detect(CHROM_A, "chr") ~ CHROM_A, TRUE ~ paste0("chr", CHROM_A))) %>%
```

Comment:
two ifelse instead of two case_when seems a lot simpler for this. I had a hard time following what this does

Reply:
At the very least say what its doing (i.e. add chr prefix to the chromosome name for builds that expect it, but only add when necessary)

Reply:
This has been updated